NLog.Web.ASPNET5.Tests/LayoutRenderers/AspNetCookieLayoutRendererTests.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Web;
+using NLog.Web.LayoutRenderers;
+using NSubstitute;
+using Xunit;
+
+namespace NLog.Web.Tests.LayoutRenderers
+{
+    public class AspNetCookieLayoutRendererTests
+    {
+        public class CookieTests
+        {
+            [Fact]
+            public void NullKeyRendersEmptyString()
```

Comment:
Looks good, maybe it's nice to create some helper methods to prevent code duplication